cli/core/pkg/cli/usage.go

```diff
@@ -5,15 +5,12 @@ package cli
 
 import (
 	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
So, for imports, go packages need to be grouped together alphabetically at the top of the import list, and github packages should follow under them.

You can run `make fmt` and it will detect and fix these automatically.